tools/run_tests/dockerize/build_interop_image.sh

```diff
@@ -37,6 +37,7 @@ set -x
 #  INTEROP_IMAGE - name of tag of the final interop image
 #  BASE_NAME - base name used to locate the base Dockerfile and build script
 #  TTY_FLAG - optional -t flag to make docker allocate tty
+#  RUNTIME - runtime setting. If specified, Dockerfile-$RUNTIME will be used.
```

Comment:
Just to help me understand, are you suggesting we flatten all the runtimes and fold it into languages?  Basically, treating all the go/grpc-matrix variants as separate languages?